spec/features/create_scan_spec.rb

```diff
@@ -55,7 +58,11 @@
 
       click_button 'Send request'
 
-      expect(page).to have_css('h1#dialogTitle', text: 'Request complete')
+      # expect(page).to have_css('h1#dialogTitle', text: 'Request complete')
```

Comment:
In case we need it again for when the illiad callback starts working?

> On May 20, 2015, at 2:45 PM, Jessie Keck <notifications@github.com> wrote:
> 
> In spec/features/create_scan_spec.rb <https://github.com/sul-dlss/sul-requests/pull/151#discussion_r30753106>:
> 
> > @@ -55,7 +58,11 @@
> >  
> >        click_button 'Send request'
> >  
> > -      expect(page).to have_css('h1#dialogTitle', text: 'Request complete')
> > +      # expect(page).to have_css('h1#dialogTitle', text: 'Request complete')
> Cruft?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/sul-dlss/sul-requests/pull/151/files#r30753106>.
>